app/screens/receive-bitcoin-screen/receive-btc.tsx

```diff
@@ -325,10 +325,12 @@ const ReceiveBtc = () => {
           }
           invoice && setInvoice(invoice)
         }
-      } catch (err) {
-        console.error(err, "error with AddInvoice")
-        crashlytics().recordError(err)
-        setErr(`${err}`)
+      } catch (err: unknown) {
+        if (err instanceof Error) {
+          console.error(err, "error with AddInvoice")
+          crashlytics().recordError(err)
+          setErr(`${err}`)
```

Comment:
setErr should probably be outside of the error check, so that we can still show a string to the screen

Reply:
There are several other places where some methods may have been unintentionally included in the conditional blocks.

Reply:
I think a PR reviewing comprehensibly the error management is probably a good way to think about it

Reply:
Typescript return either `any` or `unknown` in the catch block but I think it should typically (always?) return an Error object?